### PR TITLE
chore: edit week 1

### PR DESCRIPTION
**Pull request by troycomi**
Fixed a handful of grammatical errors.

I think I emphasized testing more than debugging, but I tried not to editorialize too much.

Since I'm editing on a terminal, I line wrapped to 80 characters in most cases.  That makes it seem like I changed a lot.

### CODE REVIEW

File: content/week1/programming_basics.ipynb [hunk 1]
```diff
@@ -407,11 +407,11 @@
     "| Mutable | Immutable |\n",
     "| --------|-----------|\n",
     "| Can be changed | Can only be replaced |\n",
-    "| Can be a key or in a set | Can't be a key or in a set |\n",
+    "| Can't be a key or in a set | Can be a key or in a set |\n",
```
Comment (troycomi) [inline content/week1/programming_basics.ipynb:410]: Pretty sure this was reversed in error.

File: content/week1/programming_basics.ipynb [hunk 2]
```diff
@@ -407,11 +407,11 @@
...
     "| `list`, `dict`, `set` | `tuple`, `str`, `int`, etc. |\n",
     "\n",
     "* A set does not have an order (so you can't use `[]` to get items from it)\n",
-    "* Dictionaries are ordered before Python 3.6 (even now you still only have direct access to keys)\n",
+    "* Dictionaries are not ordered before Python 3.6 (even now you still only have direct access to keys)\n",
```
Comment (troycomi) [inline content/week1/programming_basics.ipynb:414]: And this

File: content/week1/programming_basics.ipynb (hunk 1 above)
Comment (henryiii) [reply content/week1/programming_basics.ipynb:410]: Yep!

File: content/week1/programming_basics.ipynb (hunk 2 above)
Comment (henryiii) [reply content/week1/programming_basics.ipynb:414]: Yep!

**Comment by henryiii** [comment]
Thanks!

**Comment by henryiii** [comment]
I'll add a prettier line length setting to avoid changes in line wrapping. Don't you have an inclusive language check that would be useful, too?

**Comment by troycomi** [comment]
Yeah, https://github.com/PrincetonUniversity/blocklint
Should have pre-commit integration.  By default just checks for white/blacklist and master/slave but can be customized.

**Pull request merged**